src/core/tsi/ssl_transport_security.h

```diff
@@ -132,6 +138,7 @@ tsi_result tsi_create_ssl_server_handshaker_factory(
     size_t num_key_cert_pairs, const char* pem_client_root_certs,
     int force_client_auth, const char* cipher_suites,
     const char** alpn_protocols, uint16_t num_alpn_protocols,
+    const char* session_ticket_key, size_t session_ticket_key_size,
```

Comment:
Same as tsi_create_ssl_client_handshaker_factory. Let us not change tsi_create_ssl_server_handshaker_factory and tsi_create_ssl_server_handshaker_factory_ex. Keep them as it is (for backward compatibility). We can mark as deprecate if need.

Create a new 
`tsi_create_ssl_server_handshaker_factory_with_options(tsi_ssl_server_handshaker_options* options, tsi_ssl_server_handshaker_factory** factory);`